spanner/spec/spanner_samples_spec.rb

```diff
@@ -1338,4 +1402,175 @@ def capture
     expect(captured_output).to include "11.1212"
     expect(captured_output).to include "104.4123101"
   end
+
+  example "create backup" do
+    expect(@instance.backup(@backup_id)).to be nil
+    database = create_database_with_data
+
+    capture do
+      create_backup project_id:  @project_id,
+                    instance_id: @instance.instance_id,
+                    database_id: database.database_id,
+                    backup_id:   @backup_id,
+                    expire_time: @expire_time
+    end
+
+    expect(captured_output).to include(
+      "Backup operation in progress"
+    )
+    expect(captured_output).to include(
+      "Backup #{@backup_id} of size 607 bytes was created at"
```

Comment:
In case the size changes, I would do a regex match rather than an exact match. Or just check if backup_id is in the output text.